pkg/master/master.go

```diff
@@ -226,6 +232,17 @@ type Master struct {
 // NewEtcdStorage returns a StorageInterface for the provided arguments or an error if the version
 // is incorrect.
 func NewEtcdStorage(client tools.EtcdClient, version string, prefix string) (etcdStorage tools.StorageInterface, err error) {
+	if strings.HasPrefix(version, "exp/") {
```

Comment:
Done, though rather than pick a default version when given "" this now expects the version to valid.